mycli/key_bindings.py

```diff
@@ -61,7 +61,6 @@ def _(event):
         else:
             b.start_completion(select_first=False)
 
-    @kb.add('>', filter=vi_mode)
```

Comment:
I think the filter you want here is `vi_navigation_mode`. And then I would suggest using `]` here and `[` down below (since `>` and `<` are already bound by the default VI key bindings as part of `>>` indent and `<<` de-indent commands)